content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_select.vm

```diff
@@ -303,26 +273,21 @@ $(document).ready(function() {
 						#if($can_attach_more)
 							#if($item.isCollection())
 								#if($item.addActions && !$item.addActions.isEmpty())
-									<ul style="z-index:$actionCount;margin:0;display:block" class="makeMenu">
-										<li class="menuOpen" id="$actionCount">
-		    							&nbsp;$tlang.getString('gen.add')
-											<span class="fa fa-caret-square-o-down" aria-hidden="true"></span>
-											<span class="sr-only">$tlang.getString('gen.add')</span>
-											<ul  id="menu-$actionCount" class="makeMenuChild">
+									<ul style="z-index:$actionCount;margin:0;display:block" class="makeMenu ps-0">
+										<li class="dropdown menuOpenLi"><button type="button" class="menuOpen btn btn-primary dropdown-toggle m-2" data-bs-toggle="dropdown" aria-expanded="false">
+		    							$tlang.getString('gen.add')
+											<span class="sr-only">$tlang.getString('gen.add')</span></button>
+											<ul class="dropdown-menu">
 							        	#foreach($action in $item.addActions)
 													<li>
-														<a href="#" onclick="document.getElementById('selectedItemId').value='$item.id';document.getElementById('rt_action').value='${action.typeId}${ACTION_DELIMITER}$action.id';document.getElementById('sakai_action').value='doDispatchAction';submitform('attachForm');">
+														<a class="dropdown-item" href="#" onclick="document.getElementById('selectedItemId').value='$item.id';document.getElementById('rt_action').value='${action.typeId}${ACTION_DELIMITER}$action.id';document.getElementById('sakai_action').value='doDispatchAction';submitform('attachForm');">
```

Comment:
whenever i see href="#" i get suspicious. this should probably be a button if it's not too much work